box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/SharedLinkAccessFragment.java

```diff
@@ -55,7 +49,24 @@
 
     private View mPasswordHeader;
 
-    
+    private boolean mPasswordProtectedLinksSupported = false;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        mController.getSupportedFeatures().addOnCompletedListener(new BoxFutureTask.OnCompletedListener<BoxFeatures>() {
+            @Override
+            public void onCompleted(BoxResponse<BoxFeatures> response) {
+                if (response.isSuccess()) {
+                    mPasswordProtectedLinksSupported = response.getResult().hasPasswordProtectForSharedLinks();
+                    if (mPasswordProtectedLinksSupported && getView() != null && checkIfHasRequiredFields(mShareItem)) {
+                        setupUi();
+                    }
+                }
```

Comment:
We should probably handle the error scenario and just show all the ui anyways even if the call fails.